lib/request/TingClientRequest.php

```diff
@@ -47,6 +49,24 @@ public function getParameters() {
     return $this->parameters;
   }
 
+  public function getAuth() {
```

Comment:
I known that the rest of the code base in this repos is missing documentation, but I would like to get that corrected. So please add function documentation to these new functions as an step in the right direction.